test/elpy-insert--pip-button-value-create-test.el

```diff
@@ -12,8 +12,11 @@
     (mletf* ((executable-find
               (command)
               (cond
-                ((equal command "pip") t)
                 ((equal command "easy_install") t)))
+             (call-process
+              (program &optional infile destination display &rest args)
+               (when (and (equal program "python") (equal args '("-m" "pip"))))
```

Comment:
Oops, I forgot there is one more `--help` argument here... Could you add it, since it's already merged?